examples/testNG-single-module/src/test/java/Test1.java

```diff
@@ -0,0 +1,24 @@
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+public class Test1 {
+    @Test(groups = {"group1", "ignored"})
```

Comment:
I suggest to use `org.testng.annotations.Ignore` annotation to ignore tests (https://testng.org/doc/documentation-main.html#ignore-tests).

Reply:
Fixed